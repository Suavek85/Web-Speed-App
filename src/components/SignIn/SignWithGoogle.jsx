import React from 'react'

import { signInWithGoogle } from '../../firestore'


export default function SignWithGoogle() {
  return (
    <div  >
      <button onClick={ signInWithGoogle }>Sign in with Google</button>
    </div>
  )
}