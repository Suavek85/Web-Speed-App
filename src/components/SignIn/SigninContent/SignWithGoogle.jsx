import React from 'react'
import { signInWithGoogle } from '../../../firebase/firebase'

export default function SignWithGoogle() {
  return (
    <button 
      onClick={ signInWithGoogle }
    >
        Sign in with Google
    </button>
  )
}