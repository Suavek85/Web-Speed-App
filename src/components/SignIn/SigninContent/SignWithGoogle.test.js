import React from 'react'
import { shallow } from 'enzyme'
import SignWithGoogle from './SignWithGoogle'

describe('SignWithGoogle snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SignWithGoogle />)
    expect(wrapper).toMatchSnapshot()
  })

})