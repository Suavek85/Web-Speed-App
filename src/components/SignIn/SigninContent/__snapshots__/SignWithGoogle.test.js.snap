// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignWithGoogle snapshot should match the snapshot 1`] = `
<button
  onClick={[Function]}
>
  Sign in with Google
</button>
`;
